transit_odp/fares_validator/schema/fares_schema.json

```diff
@@ -23,6 +33,490 @@
+      "rules": [
+        {
+          "test": "count(x:Name) > 0"
+        }
+      ]
+    },
+    {
+      "details": "Mandatory element 'TypeOfFareProductRef' missing",
+      "category": "Mandatory Element",
+      "reference": "0",
+      "number": 131,
+      "context": "//x:fareProducts/x:PreassignedFareProduct",
+      "rules": [
+        {
+          "test": "count(x:TypeOfFareProductRef) > 0"
+        }
+      ]
+    },
+    {
+      "details": "Mandatory element 'ChargingMomentType' missing",
+      "category": "Mandatory Element",
+      "reference": "0",
+      "number": 131,
+      "context": "//x:fareProducts/x:PreassignedFareProduct",
+      "rules": [
+        {
+          "test": "count(x:ChargingMomentType) > 0"
+        }
+      ]
+    },
+    {
+      "details": "Mandatory element 'validableElements' missing",
+      "category": "Mandatory Element",
+      "reference": "0",
+      "number": 131,
+      "context": "//x:fareProducts/x:PreassignedFareProduct",
+      "rules": [
+        {
+          "test": "count(x:validableElements) > 0"
+        }
+      ]
+    },
+    {
+      "details": "Mandatory element 'ValidableElement' missing",
+      "category": "Mandatory Element",
+      "reference": "0",
+      "number": 131,
+      "context": "//x:fareProducts/x:PreassignedFareProduct/x:validableElements",
+      "rules": [
+        {
+          "test": "count(x:ValidableElement) > 0"
+        }
+      ]
+    },
+    {
+      "details": "Mandatory element 'fareStructureElements' missing",
+      "category": "Mandatory Element",
+      "reference": "0",
+      "number": 131,
+      "context": "//x:fareProducts/x:PreassignedFareProduct/x:validableElements/x:ValidableElement",
+      "rules": [
+        {
+          "test": "count(x:fareStructureElements) > 0"
+        }
+      ]
+    },
+    {
+      "details": "Mandatory element 'FareStructureElementRef' missing",
+      "category": "Mandatory Element",
+      "reference": "0",
+      "number": 131,
+      "context": "//x:fareProducts/x:PreassignedFareProduct/x:validableElements/x:ValidableElement/x:fareStructureElements",
+      "rules": [
+        {
+          "test": "count(x:FareStructureElementRef) > 0"
+        }
+      ]
+    },
+    {
+      "details": "Mandatory element 'accessRightsInProduct' missing",
+      "category": "Mandatory Element",
+      "reference": "0",
+      "number": 131,
+      "context": "//x:fareProducts/x:PreassignedFareProduct",
+      "rules": [
+        {
+          "test": "count(x:accessRightsInProduct) > 0"
+        }
+      ]
+    },
+    {
+      "details": "Mandatory element 'ValidableElementRef' missing",
+      "category": "Mandatory Element",
+      "reference": "0",
+      "number": 131,
+      "context": "//x:fareProducts/x:PreassignedFareProduct/x:accessRightsInProduct/x:AccessRightInProduct",
+      "rules": [
+        {
+          "test": "count(x:ValidableElementRef) > 0"
+        }
```

Comment:
Think the context should just be "//x:fareProducts/x:PreassignedFareProduct/x:accessRightsInProduct/"